airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -463,18 +463,11 @@ def _get_user_permissions(user: BaseUser):
         """
         Return the user permissions.
 
-        ACTION_CAN_READ and ACTION_CAN_ACCESS_MENU are merged into because they are very similar.
-        We can assume that if a user has permissions to read variables, they also have permissions to access
-        the menu "Variables".
```

Comment:
I'm curious, what changed so that this is no longer true? There is no description on this PR or linked issue explaining the context :grimacing:

Reply:
Short answer: backcompat